libs/cardano-ledger-test/cardano-ledger-test.cabal

```diff
@@ -54,15 +54,6 @@ library
         Test.Cardano.Ledger.Constrained.Scripts
         Test.Cardano.Ledger.Constrained.Size
         Test.Cardano.Ledger.Constrained.Solver
-        Test.Cardano.Ledger.Constrained.V2.Conway
-        Test.Cardano.Ledger.Constrained.V2.Conway.CERT
-        Test.Cardano.Ledger.Constrained.V2.Conway.DELEG
-        Test.Cardano.Ledger.Constrained.V2.Conway.GOV
-        Test.Cardano.Ledger.Constrained.V2.Conway.GOVCERT
-        Test.Cardano.Ledger.Constrained.V2.Conway.LEDGER
-        Test.Cardano.Ledger.Constrained.V2.Conway.POOL
-        Test.Cardano.Ledger.Constrained.V2.Conway.PParams
-        Test.Cardano.Ledger.Constrained.V2.Conway.UTXO
```

Comment:
Actually, here is something I do kind of take issue with. Moving these modules out of here makes it much more difficult to make changes to these constraints as you're working with them. This is because we run the tests in `Test.Cardano.Ledger.STS` and while working on these constraints one typically needs to make many small changes and repeatedly reload ghci. This becomes much more cumbersome if the modules live in a different package as every time you want to re-run a test you can no longer just `:r` but now you need to re-start ghci instead.

Reply:
I moved everything back to `cardano-ledger-test`, just changed the names of the modules a bit so they're consistent with how we name the rules in the implementation.